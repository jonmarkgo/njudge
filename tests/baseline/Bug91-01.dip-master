control   001       S1801MX   1 1 1 0 2 18 7 0 0 0 0
Process   Thu Jan  8 00:00:00 2004 (1073520000)
Start     Wed Dec 31 23:59:59 2003 (1072915199)
Ontime rat. min 0.000
T = resignation ratio max 1.000
Yet_Applied_deadline? 0
-
dt        x0        S1901M    1 1 1 -10000 10020007 18 4 0 15 0 0
Process   Thu Jan  8 00:00:00 2004 (1073520000)
Ontime rat. min 0.000
T = resignation ratio max 1.000
Yet_Applied_deadline? 0
Moves     clock 1410 min 12.00 next  71.00 grace 167.00 delay 0.50 days -MTWTF-
Retreat   clock   -1 min  0.00 next  23.00 grace  71.00 delay 0.50 days -MTWTF-
Adjust    clock   -1 min  0.00 next  23.00 grace  71.00 delay 0.50 days -MTWTF-
master       0  0  0  11 50500 pass         master1@localhost    0  0 0 0 0 0 0 0 0 0
-
