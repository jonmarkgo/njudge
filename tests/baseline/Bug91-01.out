Create Duplex game (standard for 4 players)
input/Bug91-01.txt
===== mail sent to master1@localhost/Re: XXXX:dt - S1901M 
:: Judge: XXXX  Game: Dt  Variant: Standard
:: Deadline: S1901M Thu Jan 01 1970 00:00:00 +0000  

You've been selected to play Master in game 'dt'.
You'll be notified when 7 more players sign on.
Game 'dt' now has 0 players.

Number of Players for game 'dt' set to 4.
===== end of mail ======
